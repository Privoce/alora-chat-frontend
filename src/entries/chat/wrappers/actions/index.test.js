import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	ActionsWrapper
} from 'entries/chat/wrappers';

describe('<ActionsWrapper />', () => {
	const modelI = {};

	const getWrapper = (model) => {
		return shallow((
			<ActionsWrapper {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
