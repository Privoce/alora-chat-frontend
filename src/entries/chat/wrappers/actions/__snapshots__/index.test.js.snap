// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionsWrapper /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
